sentences/de/homeassistant_HassGetCurrentDate.yaml

```diff
@@ -4,4 +4,5 @@ intents:
     data:
       - sentences:
           - "<welche> (tag|datum) (ist|haben wir) heute"
-          - "(nenn[e]|gib|gebe) mir das[ (heutige|aktuelle)] datum"
+          - "(nenn[e]|gib|gebe|sage) mir das[ (heutige|aktuelle)] datum"
+          - "was ist heute für ein tag"
```

Comment:
```suggestion
          - "was (ist|haben wir) heute für ein[en] (tag|datum)"
```
this would allow:
```yaml
- "was ist heute für ein tag"
- "was ist heute für ein datum"
- "was haben wir heute für ein tag"
- "was haben wir heute für einen tag"
- "was haben wir heute für ein datum"
```